src/main/java/frc/robot/subsystems/Climber.java

```diff
@@ -98,9 +101,21 @@ public boolean isLeftReverseLimitSwitchPressed() {
     public boolean isRightReverseLimitSwitchPressed() {
         return rightRetractedLimit.isPressed();
     }
+    public double getLeftClimberCurrent() {
```

Comment:
Why was this added? the climber is not at all an issue with regards to current draw.